testdata/google/events/cloud/cloudbuild/v1/BuildEventData-complex.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I think it would be good to have *one* source in this example - we just need to not have both `repoSource` and `storageSource`

Reply:
Done.